drivers/adc/ad7193/ad7193.h

```diff
@@ -199,6 +206,7 @@ struct ad7193_dev {
 	uint8_t			input_mode;
 	uint8_t			buffer;
 	uint8_t     		bpdsw_mode;
+	enum ad719x_adc_ver ver;
```

Comment:
nit: we usually use `chip_id` when a driver supports a familiy of devices.

Reply:
Changed ver to chip_id.